sys/conf/kern.mk

```diff
@@ -340,6 +346,7 @@ CCLDFLAGS+=	-fuse-ld=${LD:[1]:S/^ld.//1W}
 
 # Set target-specific linker emulation name.
 LD_EMULATION_aarch64=aarch64elf
+LD_EMULATION_aarch64c=aarch64elf_cheri
```

Comment:
Add an XXX that this is a temporary workaround?

Reply:
Yeah, it looks like it will stay here for some more time until we have full support in Morello lld.